packages/mds-api-server/index.ts

```diff
@@ -12,6 +12,9 @@ import {
   UserEmailClaim,
   JurisdictionsClaim
 } from '@mds-core/mds-api-authorizer'
+import promBundle from 'express-prom-bundle'
+
+const metricsMiddleware = promBundle({includeMethod: true, includePath: true, includeUp: true})
```

Comment:
I may have suggested `prometheus` and `prometheusMiddleware` ... Outside of this PR, it's not obvious what `prom` is.